torch/csrc/jit/script/compiler.cpp

```diff
@@ -1253,6 +1247,27 @@ struct to_ir {
     return emitApplyExpr(Var::create(ident.range(), ident), inputs, attributes, n_binders);
   }
 
+  std::shared_ptr<SugaredValue> emitPrint(Ident ident, const List<torch::jit::script::Expr> inputs) {
+    std::vector<std::int64_t> string_indices;
+    std::vector<std::string> strings; 
+    TreeList trees_;
+    for(size_t i = 0; i < inputs.size(); i++) {
+      if (inputs[i].kind() == TK_STRINGLITERAL) {
+        strings.push_back(StringLiteral(inputs[i]).text());
+        string_indices.push_back(i);
+      } else {
+        trees_.push_back(inputs[i]);
+      }
+    }
+    auto filtered_inputs = getNamedValues(trees_, true, identity);
+    ensureTensors(ident.range(), toValues(filtered_inputs));
```

Comment:
I can't see why is this necessary. We should be able to print pretty much anything